cfme/services/workloads.py

```diff
@@ -41,41 +95,29 @@ def __init__(self, appliance=None):
 
 @navigator.register(VmsInstances, 'All')
 class AllVMs(CFMENavigateStep):
+    VIEW = WorkloadsVM
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self, *args, **kwargs):
+        workloads = self.create_view(WorkloadsVM)
         self.prerequisite_view.navigation.select('Services', 'Workloads')
-        if 'filter_folder' not in kwargs:
-            vm_instances_tree()
-        elif 'filter_folder' in kwargs and 'filter_name' in kwargs:
-            vm_instances_tree(kwargs['filter_folder'], kwargs['filter_name'])
-        else:
-            raise DestinationNotFound("the destination isn't found")
+        workloads.search.clear_search()
+        workloads.vms.clear_filter()
 
     def am_i_here(self, *args, **kwargs):
-        if 'filter_folder' not in kwargs:
-            return match_page(summary='All VMs & Instances')
-        elif 'filter_folder' in kwargs and 'filter_name' in kwargs:
-            return match_page(summary='All VMs and Instances - '
-                                      'Filtered by "{}" ( clear )'.format(kwargs['filter_name']))
+        self.view.is_displayed()
```

Comment:
REQUIRED Why is `is_displayed` here and below a method and not a property? And basically you don't need `am_i_here` method if `is_displayed` property is implemented for a view. Look here https://github.com/ManageIQ/integration_tests/blob/master/utils/appliance/implementations/ui.py#L139